Objects/interface/UnpackedPFCand.h

```diff
@@ -13,28 +13,6 @@ namespace panda {
 
   class UnpackedPFCand : public ParticleM {
   public:
-    enum PType {
```

Comment:
Yeah I figured we don't really need to duplicate these arrays and enums for the unpacked version.